platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/idp/client/IdpLogoutActionProvider.java

```diff
@@ -76,11 +77,15 @@
       logoutUrl = new URL(logoutUrlString);
 
     } catch (MalformedURLException e) {
-      LOGGER.info("Unable to resolve URL: {}", logoutUrlString);
+      LOGGER.info("Unable to resolve URL: {}", LogSanitizer.cleanAndEncode(logoutUrlString));
     } catch (ClassCastException e) {
-      LOGGER.debug("Unable to cast parameter to Map<String, Object>, {}", subjectMap, e);
+      LOGGER.debug(
+          "Unable to cast parameter to Map<String, Object>, {}",
+          LogSanitizer.cleanAndEncode(subjectMap),
+          e);
     } catch (UnsupportedEncodingException e) {
-      LOGGER.debug("Unable to encode the encrypted timestamp.", subjectMap, e);
+      LOGGER.debug(
+          "Unable to encode the encrypted timestamp.", LogSanitizer.cleanAndEncode(subjectMap), e);
```

Comment:
The logging statement contains no `{}` so we should remove the `subjectMap` argument.